packages/docs/src/routes/docs/components/rendering/index.mdx

```diff
@@ -102,3 +109,22 @@ Most current-generation frameworks have a synchronous `render()` process. Synchr
 ## DOM update buffering
 
 The asynchronous nature of `render()` means that users may see an intermediate rendering of the UI as components download. Seeing an intermediate state is undesirable; therefore, Qwik will buffer all DOM updates and only flush the DOM operations once all of the components have been downloaded and their JSX functions executed. The result is that the UI will update as an atomic operation, and the user will not see the intermediate steps.
+
+## `<RenderOnce>`
+
+Qwik provides a Component called `<RenderOnce>`. It prevents re-renderings of nested children. Values which are connected to a state will always get updated.
```

Comment:
I think this is missing a use case
@manucorporat should this API be documented? and if so, is there a good use case for using this in apps?

Reply:
can drop that part to get the rest merged 👍 @shairez

Reply:
thanks @zanettin 
if you can, please open a separate PR for it and consider adding the comment I've written asking for a use case

I think it's valuable to know when we should consider using this so it won't confuse developers

and again, thanks for your contribution(s)! ❤

Reply:
@shairez done 🎉 🙏  new PR is here https://github.com/BuilderIO/qwik/pull/2848